src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendarSize;
+  gap: $gap;
+  padding: 10px;
+
+  @each $num, $day in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $num * ($cellSize + $gap);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} {
+      :nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__cell {
+    display: flex;
+    box-sizing: border-box;
+    @include square($cellSize);
+    border: 1px solid black;
+    background-color: #eee;
```

Create a variable for all colors
```suggestion
    background-color: $c-color;
```